README.md

```diff
@@ -66,63 +66,65 @@ Execute `sbt clean test` or `sbt clean coverage test && sbt coverageReport` for
 
 Coverage report can be seen at `target/scala-2.13/scoverage-report/index.html`
 
+### Build Docker image
+> You don't have to build the docker image yourself, as the latest master is published at
```

Comment:
Every branch and tag is posted to Docker Hub, so please, describe it more firmly.